code/script.js

```diff
@@ -33,19 +43,186 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight;
 }
 
-// Starts here
-const greetUser = () => {
+const disableChatInputAndSendButton = () => {
```

Comment:
Possibilities to create shorter function names so it's easier to read/follow (?)